images/charmed-spark/pebble/log-layer.yaml

```diff
@@ -0,0 +1,13 @@
+log-targets:
+  grafana-agent-k8s:
+    override: replace
+    type: loki
+    location: $LOKI_URL
+    services: [all]
+    labels:
+      product: charmed-spark
+      flavour: $FLAVOUR
+      app: spark
+      app_id: $SPARK_APPLICATION_ID
```

Comment:
**todo** This should be `spark_job_id` to be consistent with the label in the metrics, see [here](https://github.com/canonical/spark-k8s-bundle/blob/main/releases/3.4/resources/grafana/spark_dashboard.json#L1709)

Reply:
+1